sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -121,6 +121,8 @@ def foo((a, b)):
 
 _ANY_VAR_POSITIONAL = typehints.Tuple[typehints.Any, ...]
 _ANY_VAR_KEYWORD = typehints.Dict[typehints.Any, typehints.Any]
+# TODO(BEAM-8280): Remove this when from_callable is ready to be enabled.
+_enable_from_callable = False
```

Comment:
Nit: the name "from_callable" isn't very enlightening without context, perhaps something more descriptive like infer_types_from_annotations?

Reply:
Sorry I don't have time to make this change. I'll be away for 1.5 weeks starting tomorrow.